public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -244,6 +248,17 @@ export function RuleViewer({ match }: RuleViewerProps) {
   );
 }
 
+function GrafanaRuleUID({ rule }: { rule: GrafanaRuleDefinition }) {
```

Comment:
maybe we could pass directly the `uid` as a prop?

Reply:
I think I'd prefer to keep it this way. If we pass `uid` any `string` value will be valid, whereas passing `rule,` we force to pass `GrafanaRuleDefinition`, which minimizes the risk of error